lib/engine/game/g_18_esp/game.rb

```diff
@@ -876,11 +876,16 @@ def num_corp_trains(entity)
         end
 
         def place_home_token(corporation)
-          if corporation == mza && corporation_by_id('MZ')&.ipoed && !corporation.tokens.first.used
-            # mza special case if mz already exists on the map
+          if corporation == mza && !corporation.tokens.first.used
             token = corporation.tokens.first
             hex = hex_by_id(corporation.coordinates)
-            city = hex.tile.cities.size > 1 ? city_by_id("#{hex.tile.id}-#{corporation.city}") : hex.tile.cities.first
+            city = if corporation_by_id('MZ')&.ipoed
+                     # mza special case if mz already exists on the map
```

Comment:
In general I think the corporation abbreviations in texts and comments should use upper case, so MZA and MZ in this case. Same comment for line 886.